pipeline/r53update.py

```diff
@@ -0,0 +1,278 @@
+import argparse
```

Comment:
do you need a shabang?

Reply:
I don't put `#!/usr/bin/python` or even `#!/usr/bin/env python` because of version issues.  Some systems out there have had Python 2.6 (!) as default up until pretty recently, and the whole Python 2/3 compatibility is another matter.  Unless I spent a few more hours to ensure that something works on 2.6 through 3.7 (a painstaking task), I'd rather require users to manually run the right version of Python (3.6+ in this case, due to the use of formatted string literals).